File: wasm/bmxt-core/src/cmd/clear.rs
use crate::meta::Cmd;
use crate::model::{DispatchJson, Effect};

pub const CMD: Cmd = Cmd {
    name: "clear",
    aliases: &[],
    usage_primary: "clear",
};

pub fn run(_args: &[String]) -> DispatchJson {
    DispatchJson::effects(vec![Effect::ClearLog])
}

File: lib/features/bmxt-core/cmd/clear.ts
import type { CmdMeta } from "../types"
import { effectsDispatch } from "../types"

export const CMD: CmdMeta = {
  name: "clear",
  aliases: [],
  usagePrimary: "clear"
}

export function run(_args: string[]) {
  return effectsDispatch([{ kind: "clear_log" }])
}
